lib/easypost/internal_utilities.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module EasyPost::InternalUtilities
+end
+
+require_relative 'utilities/json'
+require_relative 'utilities/system'
```

Comment:
issue: require statements should be at the top of a file, not the end

Reply:
Moving these `require_relative` on the top will throw `uninitialized constant`. This follows the pattern for the current `beta` module: https://github.com/EasyPost/easypost-ruby/blob/master/lib/easypost/beta.rb

This applies to the rest comments.

Reply:
suggestion: it may be good to leave a comment above the relative import lines to describe why they come later